packages/utilities/psammead-styles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-styles",
-  "version": "7.3.0",
+  "version": "10.0.0",
```

Comment:
Should be minor as discussed elswhere I think:
https://github.com/bbc/psammead/pull/4545#discussion_r708161813